samples/DirectX/D3D12/HelloTexture12.cs

```diff
@@ -0,0 +1,929 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
```

Comment:
I can't comment on the actual file, but `HelloOutput.png` shouldn't be checked in. Checking in binary files generally isn't a good idea and while a few small sample pictures might be okay, a 60kb file isn't ideal and is a lot larger than would be liked.

I'd rather not check it in at all and look at alternatives for hosting any sample pictures.

Reply:
Ah OK. I'll remove the reference to it. It's not essential.